from src.exception import CustomException
print("Import successful!")
